operator/v1/0000_50_ingress-operator_00-ingresscontroller.crd.yaml

```diff
@@ -1202,6 +1202,22 @@ spec:
                       are subject to change over time."
                     format: duration
                     type: string
+                  maxConnections:
+                    description: "maxConnections defines the maximum number of simultaneous
+                      connections that can be established per HAProxy process. Increasing
+                      this value allows each ingress controller pod to handle more
+                      connections but at the cost of additional system resources being
+                      consumed. \n If this field is empty or 0, the IngressController
+                      will use the default value of 20000, but the default is subject
+                      to change in future releases. If the value is -1 then HAProxy
+                      will dynamically compute a value based on ulimits and the the
+                      configuration file when the haproxy process starts. Selecting
+                      -1 (i.e., auto) will result in a large value being computed
+                      (typically 524259 on OpenShift >=4.10 clusters) and therefore
```

Comment:
Is it really 524259 and not 524288 (= 2<sup>19</sup>)?

Reply:
> Is it really 524259 and not 524288 (= 219)?

Depends on what's in your haproxy.config. Maybe we should just quote a more generic number. 520000?